src/routes/news/components/__tests__/LocalNewsList.spec.js

```diff
@@ -0,0 +1,52 @@
+// @flow
+
+import React from 'react'
+import { LocalNewsModel } from '@integreat-app/integreat-api-client'
+import moment from 'moment-timezone'
+import { shallow } from 'enzyme'
+import LocalNewsList from '../LocalNewsList'
+import NewsElement from '../NewsElement'
+import { LOCAL_NEWS } from '../../constants'
+
+describe('LocalNewsList', () => {
+  const language = 'en'
+  const path = '/testumgebung/en/news/local'
+  const t = (key: ?string): string => key || ''
+  const city = 'testcity'
+
+  const renderItem = ({ id, title, message, timestamp }, city) => <NewsElement
+    id={id}
+    title={title}
+    content={message}
+    timestamp={timestamp}
+    type={LOCAL_NEWS}
+    key={localNews1.id}
+    path={path}
+    t={t}
+    language={language}
+  />
+  const date = moment.tz('2020-03-20 17:50:00', 'GMT')
+  const localNews1 = new LocalNewsModel({
+    id: 217,
+    title: 'Tick bite - What to do?',
+    timestamp: date,
+    message:
+    'In summer there are often ticks in forest and meadows with high grass. These are very small animals. They feed on the blood of people or animals they sting, like mosquitoes. But they stay in the skin longer and can transmit dangerous diseases. If you have been in high grass, you should search your body very thoroughly for ticks. They like to sit in the knees, armpits or in the groin area. If you discover a tick in your skin, you should carefully pull it out with tweezers without crushing it. If the sting inflames, you must see a doctor.'
+  })
+
+  const localNews2 = new LocalNewsModel({
+    id: 218,
+    title: 'Tick bite - What to do?',
+    timestamp: date,
+    message:
+    'In summer there are often ticks in forest and meadows with high grass. These are very small animals. They feed on the blood of people or animals they sting, like mosquitoes. But they stay in the skin longer and can transmit dangerous diseases. If you have been in high grass, you should search your body very thoroughly for ticks. They like to sit in the knees, armpits or in the groin area. If you discover a tick in your skin, you should carefully pull it out with tweezers without crushing it. If the sting inflames, you must see a doctor.'
+  })
+
+  const items = [localNews1, localNews2]
+
+  it('should render and match snapshot', () => {
```

Comment:
It would be very good if this tests would assert fro actual behaviour of the component. Like that it displays some items.

Reply:
Should be fixed. I added a couple more test cases.

Reply:
@amusameh I think you can remove this snapshot tests as it doesn't really test anything